parquet_go18_test.go

```diff
@@ -121,6 +121,67 @@ func ExampleWrite_any() {
 	// map["FirstName":"R2" "LastName":"D2"]
 }
 
+func ExampleSearch() {
+	type Row struct{ FirstName, LastName string }
+
+	buf := new(bytes.Buffer)
+	// The column being searched must be sorted for search to work.
```

Comment:
My first question reading this comment is, how is the column sorted? First name? Last name? Is this specified somewhere?

Reply:
I had the same question, so I should probably add some clarity here once I know the proper way! 😅